actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -84,7 +84,8 @@ def controller_class
         PASS_NOT_FOUND
       end
     end
-
+    #key? method returns a boolean value
+    #returns true if 'key'(the parameter) has been set accordingly
```

Comment:
A couple of style tweaks
* The space should be added back - we don't want the documentation touching the line after end.
* There should be spaces between the pound and the documentation for example: `# key? method...`
* I think this documentation would be better written as: 

```
# Returns true if the request has set a header matching the given key parameter
#
#    request.key? :ip_spoofing_check # => true
```

When you've done that can you squash your commits into one? Thanks!